spec/system/courses/review_spec.rb

```diff
@@ -236,6 +236,37 @@
       end
     end
 
+    scenario 'coach changes the sort order of submissions', js: true do
+      sign_in_user course_coach.user, referer: review_course_path(course)
+
+      within("div[aria-label='change-submissions-sorting']") do
+        expect(page).to have_content("Submitted At")
+      end
+
+      # Check current ordering of pending items
+      find("div#pending-submissions a:nth-child(1)").should have_content(submission_l1_t1.title)
+      find("div#pending-submissions a:nth-child(2)").should have_content(submission_l2_t2.title)
+      find("div#pending-submissions a:nth-child(3)").should have_content(submission_l3_t3.title)
```

Comment:
There is no need for the `div` prefix in these selectors - that's the default tag.

```
find("#pending-submissions a:nth-child(1)").should have_content(submission_l1_t1.title)
```